lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = JSON.parse(
+	fsT.readFileSync('lib/js/importers/configs/allInOneConfig.json', 'utf8')
+);
+
+interface IResultJsonObject {
+	id: string;
+	label: string;
+	value: string;
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonSource: { data }
+) => {
+	let hexToCssVariable: Array<Object> = [];
+	try {
+		let result: Array<string> = [];
+		let resultJson = {};
+
+		jsonSource.data.colors.forEach(obj => {
+			const patternIsRawColor = /RAW\/|theme/i;
+			const patternDeprecated = /deprecated|Pattern/i;
+			const patternTheme = /theme/i;
+			if (obj.name.match(patternIsRawColor) && obj.name.match(patternDeprecated) === null) {
+				const nameSplit = obj.name.split('/');
+				let colorName = (nameSplit[2] === undefined) ? nameSplit[1] : nameSplit[2];
+
+				obj.values.hex = obj.values.hex.replace(/FFFFFF/gi, function(){
+					return 'fff';
+				});
+
+				const colorPrefix = obj.name.match(patternTheme) ? '--' : '--raw-';
+
+				if (hexToCssVariable[obj.values.hex] === undefined) {
+					hexToCssVariable[obj.values.hex] = colorPrefix + colorName;
```

Comment:
we use `colorPrefix + colorName` 3 times, maybe we could create a separate variable for it?

Reply:
Done.